erpnext/projects/doctype/project/project.json

```diff
@@ -1269,12 +1273,32 @@
    "fieldtype": "Link",
    "label": "Applies to Serial No",
    "options": "Serial No"
+  },
+  {
+   "fieldname": "sec_service_feedback",
+   "fieldtype": "Section Break",
+   "label": "Service Feedback"
+  },
+  {
+   "fieldname": "feedback_date",
+   "fieldtype": "Date",
+   "label": "Feedback Date"
+  },
+  {
+   "fieldname": "feedback_time",
+   "fieldtype": "Time",
+   "label": "Feedback Time"
+  },
+  {
+   "fieldname": "feedback_remark",
```

Comment:
lets change the fieldname to something like `customer_feedback`? and all variable names as well.